tutorials/cbt-import-csv.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Import a CSV file into a Cloud Bigtable table
+description: Learn how to import a CSV file into a Cloud Bigtable table.
+author: thebilly
+tags: Cloud Bigtable, Dataflow, Java
+date_published: 2018-06-26
+---
+
+
+
+## Prerequisites
+
+### Install software and download sample code
+
+
+Make sure you have the following software installed:
+
+- [Git](https://help.github.com/articles/set-up-git/)
+- [Java SE 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Apache Maven 3.3.x or later](https://maven.apache.org/install.html)
+       
+    > If you haven't used Maven before check out this [5 minute quickstart](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html)
+
+### Set up your Cloud Project
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. Enable billing for your project.
+1.  Install the **[Google Cloud SDK](https://cloud.google.com/sdk/)** if you do
+    not already have it. Make sure you
+    [initialize](https://cloud.google.com/sdk/docs/initializing) the SDK. Use
+    your project's ID to set the default project for the `gcloud` command-line tool. 
+
+## Upload your CSV
+
+### Create a Cloud Storage bucket
+
+You use Cloud Storage to store your application's dependencies. Feel free to use an existing bucket if you have one.
```

Comment:
dependencies : not clear here, what that means. But I see that its where the data to be imported lives. might be worth clarifying.
"You use" can be removed here. And you might want to also `s/Cloud Storage/ Google Cloud Storage/`. Same goes for "Cloud Project" in the above title.

Reply:
Jeff corrected me from using Google Cloud Storage in that heading earlier actually. @hegemonic are there guidelines for when to say Google Cloud Storage vs just Cloud Storage?

Reply:
The current guidance is that we should never include "Google" in the name of a GCP product, with the exception of "Google Cloud Platform." (There are still lots of pages on cloud.google.com that haven't been updated based on this guidance, which is confusing, I'll admit.)

Regarding the last part of Jisha's comment, I believe that the correct term for a project is either "Google Cloud Platform project" (if you haven't already used "Google Cloud Platform" earlier in the page) or "GCP project" (if you have).